### PR TITLE
Hospitality Assistant Updates

### PR DESCRIPTION
**Pull request by lauren-mills**
### Purpose
*What is the context of this pull request? Why is it being done?*
- Update to latest VA sample
- Updated Chitchat and FAQ
- Adds in GoogleAdapter support/responses

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: samples/csharp/assistants/hospitality-assistant/Bot.Builder.Community.Adapters.Google/GoogleConversationalSchema.cs.old [hunk 1]
```diff
@@ -0,0 +1,399 @@
+﻿namespace Bot.Builder.Community.Adapters.Google
```
Comment (lzc850612) [inline samples/csharp/assistants/hospitality-assistant/Bot.Builder.Community.Adapters.Google/GoogleConversationalSchema.cs.old]: do we need this file?

File: samples/csharp/assistants/hospitality-assistant/Bot.Builder.Community.Adapters.Google/GoogleHelper.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        }
+
+        internal static bool ValidateRequest(HttpRequest httpRequest, string actionProjectId)
+        {
+            return GetProjectIdFromRequest(httpRequest).ToLowerInvariant() == actionProjectId.ToLowerInvariant();
+        }
+
+        internal static string StripInvocation(string query, string invocationName)
+        {
+            if (query != null && (query.ToLower().StartsWith("talk to") || query.ToLower().StartsWith("speak to")
```
Comment (lzc850612) [inline samples/csharp/assistants/hospitality-assistant/Bot.Builder.Community.Adapters.Google/GoogleHelper.cs:34]: the coding style here looks odd :)

**Review by lzc850612 — CHANGES_REQUESTED** [review]

**Comment by lzc850612** [comment]
@pavolum , i understand the reason of needing the GoogleAdapter source code is because you had to do manual changes in there. can we take a look at what's the change needed and come up with an approach to extend it instead of having to include the full source code? don't have to do it now but need to create an issue to track and fix it later.

File: samples/csharp/assistants/hospitality-assistant/Bot.Builder.Community.Adapters.Google/GoogleConversationalSchema.cs.old (hunk 1 above)
Comment (pavolum) [reply samples/csharp/assistants/hospitality-assistant/Bot.Builder.Community.Adapters.Google/GoogleConversationalSchema.cs.old]: Explanation below in PR thread

**Comment by pavolum** [comment]
> @pavolum , i understand the reason of needing the GoogleAdapter source code is because you had to do manual changes in there. can we take a look at what's the change needed and come up with an approach to extend it instead of having to include the full source code? don't have to do it now but need to create an issue to track and fix it later.

Yeah this is what I wanted to mention at the meeting. This is by no means what the final implementation of the adapter within the hospitality skill. I have just gotten a bit hacky so we can branch off this initial implementation and develop the demo for tomorrow. I have a list of changes I will make before its truly done. Ill make a task for that just to remind me. The list is as follows:

- Remove local build of google adapter and link to the most up to date and stable google adapter NuGet package (we shouldn't need an extension as my changes have been implemented in the adapter itself)
- Remove all provider conditional logic in the code and move them to .lg (I have validated the possibility of this but it gets a bit trickier with google card responses)
- Add Alexa adapter
- Add documentation on best practices when using the VA with the adapters

File: samples/csharp/assistants/hospitality-assistant/Bot.Builder.Community.Adapters.Google/GoogleHelper.cs (hunk 2 above)
Comment (lauren-mills) [reply samples/csharp/assistants/hospitality-assistant/Bot.Builder.Community.Adapters.Google/GoogleHelper.cs:34]: @pavolum

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**